packages/api/src/command/medical/patient/reset-external-data.ts

```diff
@@ -0,0 +1,42 @@
+import { MedicalDataSource, isMedicalDataSource } from "@metriport/core/external/index";
+import { BadRequestError } from "@metriport/shared";
+import { PatientModel } from "../../../models/medical/patient";
+import { getPatientModelOrFail } from "./get-patient";
+import { executeOnDBTx } from "../../../models/transaction-wrapper";
+
+export type ResetExternalData = {
+  cxId: string;
+  patientId: string;
+  source: string;
```

Comment:
This should be of type `MedicalDataSource`. no?

Reply:
It comes from a param so id have to alias the param in order to add MedicalDataSource